setup.py

```diff
@@ -60,7 +60,11 @@ def find_version(*file_paths):
         'snakemake',
     ],
     extras_require={
-        'testing': ['pytest-flake8']
+        'testing': ['pytest-flake8'],
+        'builtins': [
```

Comment:
Either we should move these requirements to install_requires or we should make it so that qg can still run without them installed (and we throw an error if they aren't installed and someone tries to use that builtin)